src/sensorpush_ble/parser.py

```diff
@@ -16,21 +16,25 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-SENSORPUSH_DEVICE_TYPES = {64: "HTP.xw", 65: "HT.w"}
+SENSORPUSH_DEVICE_TYPES = {1: "HT1", 64: "HTP.xw", 65: "HT.w"}
 
 SENSORPUSH_MANUFACTURER_DATA_LEN = {
     3: "HT.w",
     5: "HTP.xw",
 }
 
-LOCAL_NAMES = ["HTP.xw", "HT.w"]
+LOCAL_NAMES = ["s", "HT1", "HTP.xw", "HT.w"]
+
+SENSORPUSH_UUID = "ef090000-11d6-42ba-93b8-9dd7ec090aa9"
+SENSORPUSH_UUID_V2 = "ef090000-11d6-42ba-93b8-9dd7ec090ab0"
 
```

Comment:
Please name the constants to show intent

https://en.wikipedia.org/wiki/Magic_number_(programming)

> Names chosen to be meaningful in the context of the program can result in code that is more easily understood by a maintainer who is not the original author (or even by the original author after a period of time).[[5]](https://en.wikipedia.org/wiki/Magic_number_(programming)#cite_note-Paul_2002_SYMBOLS-5) An example of an uninformatively named constant is int SIXTEEN = 16, while int NUMBER_OF_BITS = 16 is more descriptive.

Reply:
Can you let me know what you would like these values to be? I am just trying to help here, and I'm not aware of all the contribution requirements for your repo

Reply:
new names LGTM